src/routes/github/repository/github-repository-get.ts

```diff
@@ -62,7 +62,7 @@ const getReposBySubscriptions = async (repoName: string, subscriptions: Subscrip
 				createInstallationClient(subscription.gitHubInstallationId, jiraHost, metrics, logger, subscription.gitHubAppId)
 			]);
 
-			const searchQueryInstallationString = `${repoName} org:${orgName} in:full_name`;
+			const searchQueryInstallationString = `${repoName} org:${orgName} in:full_name fork:true`;
```

Comment:
Would it still show regular repos?

Reply:
Yup, it should. Tested it quickly locally.
As per the [documentation](https://docs.github.com/en/search-github/searching-on-github/searching-in-forks), there are two options for `fork`:
- `fork:true` that matches all repositories containing the org name including forks.
- `fork:only`that matches all fork repositories containing the org name